CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## TS CLI&SDK [2.1.3](https://github.com/marinade-finance/validator-bonds/compare/v2.1.2...v2.1.3) (2025-02-20)
```

Comment:
(non-blocking)
The last tag in this repo that GH shows me is `v2.0.6`, if these diffs are expected to be used by the readers the tags should be added.

Reply:
Thank you. Good catch. There is not an automation for the release notes and I need to do that manually.
It would be nice to have it in place for github pipeline but I haven't got time for it.